stl/inc/sstream

```diff
@@ -326,6 +439,41 @@ protected:
         _Mystate = _State;
     }
 
+#if _HAS_CXX20
+    void _Init_inplace(_Mystr&& _Str, int _State) {
+        if ((_State & _Constant) && !(_State & _Noread)) { // readonly, copy the string
+            _Init(_Str.c_str(), _Str.size(), _State);
+        } else { // writeable, construct in place
+            auto _Buffer = _Str._Release_to_buffer();
+            _Init_inplace(_STD get<0>(_Buffer), _STD get<1>(_Buffer), _STD get<2>(_Buffer), _State);
+        }
+    }
+
+    // Call this method only when writeable
+    void _Init_inplace(_Elem* _Pnew, _Mysize_type _Length, _Mysize_type _Count, int _State) {
+        if (_Count != 0
+            && (_State & (_Noread | _Constant))
+                   != (_Noread | _Constant)) { // finite buffer that can be read or written, set it up

```

Comment:
The wrapping here is pretty hard to read; can this comment be "detached" from the brace, and put on a line following the brace? (We've been fixing examples of this throughout the existing codebase; fallout from our clang-format conversion.)